packages/experimental/Avatar/src/useAvatar.ts

```diff
@@ -37,9 +40,31 @@ export const useAvatar = (props: JSAvatarProps): AvatarInfo => {
       activeAppearance,
       image: imageProps,
       badge: badgeProps,
+      initials: _initials,
     },
     state: {
       ...state,
     },
   };
 };
+
+export const getInitials = (name: string): string => {
+  if (!name) return '';
+  const nonWordRegExp = /\W+(?<!\S-)/g;
```

Comment:
I think it'd be more descriptive to say "nonAlphanumericRegExp"

Also, wouldn't we want to strip numerics?

Reply:
Ahh right ok

Reply:
I added validation to getInitials. Because anyway if name is not valid, initials would be an empty string